onnxruntime/core/providers/nv_tensorrt_rtx/nv_execution_provider.cc

```diff
@@ -1451,32 +1467,38 @@ std::unique_ptr<IndexedSubGraph> NvExecutionProvider::GetSubGraph(SubGraph_t gra
             erased.insert(output);
           } else if (erased.find(output) == erased.end()) {
             if (graph_output_names.find(output->Name()) != graph_output_names.end()) {
-              graph_outputs_to_add[output] = output_order;
+              // This output is the graph's output.
+              // So the output should be put into the subgraph's output list.
+              graph_outputs_to_add.insert({output, output_order});
```

Comment:
> graph_outputs_to_add.insert({output, output_order}); [](http://example.com/codeflow?start=13&length=53)

It would not insert as before if the entry existed